app/Platform/Models/Game.php

```diff
@@ -85,6 +85,7 @@ class Game extends BaseModel implements HasComments, HasMedia
         'IsFinal',
         'RichPresencePatch',
         'GuideURL',
+        'Updated',
```

Comment:
Does it still have to be exposed or is it still used?

Reply:
Indeed, still necessary.